README.md

```diff
@@ -44,15 +44,15 @@ $ npm run client-dev
 > todoboilerplate@0.0.1 client-dev /home/eric/JS/todo_boilerplate
 > webpack-dev-server --colors --hot --inline --host 0.0.0.0 --port 8080
 
-http://0.0.0.0:8080/
+http://127.0.0.1:8877/
```

Comment:
0.0.0.0 = empty mask => open to the outside
127.0.0.1 = localhost => only me

Opening a project that's still under development to the outside is considered bad practice due to the fast iteration cycle potentially leading to more chances of a vulnerability.